src/main/java/hudson/plugins/jacoco/JacocoPublisher.java

```diff
@@ -34,348 +44,445 @@
     /**
      * {@link hudson.model.HealthReport} thresholds to apply.
      */
     public JacocoHealthReportThresholds healthReports;
 
-    
+
     /**
      * Variables containing the configuration set by the user.
      */
-	private final String execPattern;
-	private final String classPattern;
-	private final String sourcePattern;
-	private final String inclusionPattern;
-	private final String exclusionPattern;
-	
-    
-    private final String minimumInstructionCoverage;
-    private final String minimumBranchCoverage;
-    private final String minimumComplexityCoverage;
-    private final String minimumLineCoverage;
-    private final String minimumMethodCoverage;
-    private final String minimumClassCoverage;
-    private final String maximumInstructionCoverage;
-    private final String maximumBranchCoverage;
-    private final String maximumComplexityCoverage;
-    private final String maximumLineCoverage;
-    private final String maximumMethodCoverage;
-    private final String maximumClassCoverage;
-    private final boolean changeBuildStatus;
-    
-
-	/**
+    private String execPattern;
+    private String classPattern;
+    private String sourcePattern;
+    private String inclusionPattern;
+    private String exclusionPattern;
+
+
+    private String minimumInstructionCoverage;
+    private String minimumBranchCoverage;
+    private String minimumComplexityCoverage;
+    private String minimumLineCoverage;
+    private String minimumMethodCoverage;
+    private String minimumClassCoverage;
+    private String maximumInstructionCoverage;
+    private String maximumBranchCoverage;
+    private String maximumComplexityCoverage;
+    private String maximumLineCoverage;
+    private String maximumMethodCoverage;
+    private String maximumClassCoverage;
+    private boolean changeBuildStatus;
+
+    @DataBoundConstructor
+    public JacocoPublisher() {
+        this.execPattern = "**/**.exec";
+        this.classPattern = "**/classes";
+        this.sourcePattern = "**/src/main/java";
+        this.inclusionPattern = "";
+        this.exclusionPattern = "";
+        this.minimumInstructionCoverage = "0";
+        this.minimumBranchCoverage = "0";
+        this.minimumComplexityCoverage = "0";
+        this.minimumLineCoverage = "0";
+        this.minimumMethodCoverage = "0";
+        this.minimumClassCoverage = "0";
+        this.maximumInstructionCoverage = "0";
+        this.maximumBranchCoverage = "0";
+        this.maximumComplexityCoverage = "0";
+        this.maximumLineCoverage = "0";
+        this.maximumMethodCoverage = "0";
+        this.maximumClassCoverage = "0";
+        this.changeBuildStatus = false;
+    }
+
+    /**
      * Loads the configuration set by user.
      */
-    @DataBoundConstructor
+    @Deprecated
     public JacocoPublisher(String execPattern, String classPattern, String sourcePattern, String inclusionPattern, String exclusionPattern, String maximumInstructionCoverage, String maximumBranchCoverage
```

Comment:
The next several hundred lines of diff are nothing but formatting changes. Also, it looks like there's some reordering of methods going on as well.